research/object_detection/utils/variables_helper.py

```diff
@@ -42,7 +42,7 @@ def filter_variables(variables, filter_regex_list, invert=False):
     a list of filtered variables.
   """
   kept_vars = []
-  variables_to_ignore_patterns = filter(None, filter_regex_list)
+  variables_to_ignore_patterns = list(filter(None, filter_regex_list))
```

Comment:
I know this is an old issue, but I think this PR can be closed.
(it is ok with only the filter -  it would create a filter object and you can iterate through it - it gives you back the next element)

Reply:
Ok, sounds good :+1:.